src/globals.js

```diff
@@ -40,7 +40,7 @@ function joiner(sep) {
 function globals() {
     return {
         range: function(start, stop, step) {
-            if(!stop) {
+            if(!stop && stop !== 0) {
```

Comment:
Range isn't documented as accepting `null` or `false` for these arguments, and there's no reason to pass them. The purpose of this clause is just to handle the case where only one or two arguments are provided, and in that case the not-provided arguments are `undefined`.